src/util/product-request.js

```diff
@@ -1,87 +1,158 @@
-		headers: {
-			authorization: authToken
-		}
-	});
-
-const addToWishList = async (authToken, product) =>
-	await axios.post(
-		API_WISHLIST,
-		{ product },
-		{
+const getWishlist = async authToken => {
+	try {
+		const data = await axios.get(API_WISHLIST, {
 			headers: {
 				authorization: authToken
 			}
-		}
-	);
+		});
+		return data;
+	} catch (err) {
+		notify(error, 'Unable to fetch data');
+		console.error(err);
+	}
+};
+
+const addToWishList = async (authToken, product) => {
+	try {
+		const data = await axios.post(
+			API_WISHLIST,
+			{ product },
+			{
+				headers: {
+					authorization: authToken
+				}
+			}
+		);
+		notify(success, 'Added to Wishlist');
+		return data;
+	} catch (err) {
+		notify(error, 'Unable to post data');
+		console.error(err);
+	}
+};
 
-const removeFromWishList = async (authToken, product) =>
-	await axios.delete(`${API_WISHLIST}/${product._id}`, {
-		headers: {
-			authorization: authToken
-		}
-	});
+const removeFromWishList = async (authToken, product) => {
+	try {
+		const data = await axios.delete(`${API_WISHLIST}/${product._id}`, {
+			headers: {
+				authorization: authToken
+			}
+		});
+		notify(success, 'Removed from Wishlist');
+		return data;
+	} catch (err) {
+		notify(error, 'Unable to remove data');
+		console.error(err);
+	}
+};
 
 // API functions for Cart
-const getCart = async authToken =>
-	await axios.get(API_CART, {
-		headers: {
-			authorization: authToken
-		}
-	});
 
-const addToCart = async (authToken, product) =>
-	await axios.post(
-		API_CART,
-		{ product },
-		{
+const getCart = async authToken => {
+	try {
+		const data = await axios.get(API_CART, {
 			headers: {
 				authorization: authToken
 			}
-		}
-	);
-
-const removeFromCart = async (authToken, product) =>
-	await axios.delete(`${API_CART}/${product._id}`, {
-		headers: {
-			authorization: authToken
-		}
-	});
+		});
+		return data;
+	} catch (err) {
+		notify(error, 'Unable to fetch data');
+		console.error(err);
+	}
+};
 
-const changeCartQuantity = async (authToken, product, type) =>
-	await axios.post(
-		`${API_CART}/${product._id}`,
-		{
-			action: {
-				type
+const addToCart = async (authToken, product) => {
+	try {
+		const data = await axios.post(
+			API_CART,
+			{ product },
+			{
+				headers: {
+					authorization: authToken
+				}
 			}
-		},
-		{
+		);
+		notify(success, 'Added to Cart');
+		return data;
+	} catch (err) {
+		notify(error, 'Unable to post data');
+		console.error(err);
+	}
+};
+
+const removeFromCart = async (authToken, product) => {
+	try {
+		const data = await axios.delete(`${API_CART}/${product._id}`, {
 			headers: {
 				authorization: authToken
 			}
-		}
-	);
+		});
+		notify(success, 'Removed from Cart');
+		return data;
+	} catch (err) {
+		notify(error, 'Unable to remove data');
+		console.error(err);
+	}
+};
+
+const changeCartQuantity = async (authToken, product, type) => {
+	try {
+		const data = await axios.post(
+			`${API_CART}/${product._id}`,
```

Comment:
Great you have saved the API path in a variable. What you can do you can save it as environmental variables.

Reply:
Thanks for the suggestion. Will keep that as a future optimization